Svc/FileUplink/FileUplink.hpp

```diff
@@ -171,17 +171,17 @@ namespace Svc {
 
           //! Record a warning
           void warning() {
-            ++this->n;
-            this->fileUplink->tlmWrite_Warnings(n);
+            ++this->m_n;
+            this->m_fileUplink->tlmWrite_Warnings(m_n);
           }
 
         PRIVATE:
 
           //! The total number of warnings
-          U32 n;
```

Comment:
Another minor nit, not blocking for this PR....but given this PR is changing much, could we rename this to `m_warning_count` or something more descriptive?

Reply:
I changed the `m_n` variable names to something more descriptive, and updated the comments of the variables. Let me know if there is a mistake in the comments